enable-and-configure-rbac.groovy

```diff
@@ -0,0 +1,333 @@
+import jenkins.model.Jenkins
+import hudson.security.Permission
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationStrategyImpl
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationConfig
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationPlugin
+import nectar.plugins.rbac.groups.Group
+import nectar.plugins.rbac.groups.GroupContainer
+import nectar.plugins.rbac.groups.GroupContainerLocator
+import nectar.plugins.rbac.groups.RootProxyGroupContainer
+import nectar.plugins.rbac.roles.Role
+import nectar.plugins.rbac.roles.Roles
+import java.io.IOException
+import org.yaml.snakeyaml.Yaml
+/*
+ * Some important lessons I learned writing this code:
+ * 1. This is a cached repository https://repo.cloudbees.com/content/repositories/dev-connect
+ *    So if the version of the plugin you are looking for does not appear, you need to try and type it into the URL
+ *    directly, and as long as you get it correct, the plugin and it's files should appear as needed.
+ * 2. From the Operations Center you need to find out what version of the
+ *    Operations Center Server Role Based Access Control plugin is installed.
+ *    Here are the releases https://release-notes.cloudbees.com/product/58
+ * 3. Then you will need to look up the pom.xml for that version of the plugin here
+ *    https://repo.cloudbees.com/content/repositories/dev-connect/com/cloudbees/operations-center/server/operations-center-rbac/
+ *    to find what version of the nectar-rbac plugin is being used. (Which contains the nectar.plugin.rbac classes)
+ * 4. You can then go here to download the javadocs for the plugin to see how it's changed when this code breaks
+ *    https://repo.cloudbees.com/content/repositories/dev-connect/com/cloudbees/nectar/nectar-rbac/
+ *
+ * This script has been verified to work on Cloudbees Core 2.190.2.2 and nectar-rbac 5.25
```

Comment:
:+1: 

But put the description at top including all these fields https://github.com/cloudbees/jenkins-scripts/blob/master/set-License-activation.groovy#L1-L10

Reply:
Sure will do